docs/users/customization-extendibility.md

```diff
@@ -79,5 +79,14 @@ services:
       - solr:$DDEV_HOSTNAME
 ```
 
+## Providing custom nginx configuration
+The default web container for ddev uses NGINX as the web server. A default configuration is provided in the web container that should work for most Drupal 7+ and WordPress sites. Some sites may require custom configuration, for example to support a module or plugin requiring special rules. To accommodate these needs, ddev provides a way to replace the default configuration with a custom version.
+
+- Run `ddev config` for the site if it has not been used with ddev before.
+- Create a file named "nginx-site.conf" in the ".ddev" directory for your site.
+- Add your configurations to the "nginx-site.conf" file. You can optionally use the [default NGINX configuration provided by ddev](https://github.com/drud/docker.nginx-php-fpm-local/blob/master/files/etc/nginx/nginx-site.conf) as a reference or starting point. [Additional configuration examples](https://www.nginx.com/resources/wiki/start/#other-examples) and documentation are available at the [nginx wiki](https://www.nginx.com/resources/wiki/)
```

Comment:
Link here is broken, https://github.com/drud/docker.nginx-php-fpm-local/blob/master/files/etc/nginx/nginx-site.conf, but I see it's only *temporarily* broken until that PR goes in.